### DISCUSSION TITLE
20240610: Web radios no longer work

### DISCUSSION THREAD

**Discussion by sibezwo** [rern/rAudio#1686] [Fixed Issues]
After the update 20240610, all my streams show "Station not reachable" even though foobar2000 plays them without any problems. For example:

http://185.33.21.111:80/smoothjazz_mobile_mp3

http://144.217.158.59:5270/stream

Raspberry Pi 4B 4GB
IQaudIO DAC HiFi pcm512x

Thomas

**Comment by ceejayemm** [comment]
Sounds similar to my #1682 

Chris

**Comment by sibezwo** [comment]
Thanks for fixing this problem!

With update 20240612, everything is working as before.

Thomas
